org.eclim.jdt/java/org/eclim/plugin/jdt/command/doc/GetElementDocCommand.java

```diff
@@ -65,7 +65,8 @@
     "OPTIONAL o offset ARG," +
     "OPTIONAL l length ARG," +
     "OPTIONAL e encoding ARG," +
-    "OPTIONAL u url ARG"
+    "OPTIONAL u url ARG," +
+    "OPTIONAL h htmlFlag ARG"
```

Comment:
Can you also make this a `NOARG`. There is no reason to supply a value for this case. The presence of the flag is enough.